client/interfaces.go

```diff
@@ -161,6 +161,8 @@ type ContainerServer interface {
 	UpdateStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePut, ETag string) (err error)
 	DeleteStoragePoolVolume(pool string, volType string, name string) (err error)
 	RenameStoragePoolVolume(pool string, volType string, name string, volume api.StorageVolumePost) (err error)
+	CopyStoragePoolVolume(source ContainerServer, pool string, volume api.StorageVolumesPost, args *StoragePoolVolumeCopyArgs) (op *RemoteOperation, err error)
+	MoveStoragePoolVolume(source ContainerServer, pool string, volume api.StorageVolumesPost, args *StoragePoolVolumeCopyArgs) (*RemoteOperation, error)
```

Comment:
All arguments and return values in interfaces.go should be named. This one is missing "op" and "err".

Reply:
To make those two line up better with the existing copy functions, they should become:

```go
CopyStoragePoolVolume(pool string, source ContainerServer, sourcePool string, volume api.StorageVolume, args *StoragePoolVolumeCopyArgs) (op *RemoteOperation, err error)
MoveStoragePoolVolume(pool string, source ContainerServer, sourcePool string, volume api.StorageVolume, args *StoragePoolVolumeMoveyArgs) (op *RemoteOperation, err error)

type StoragePoolVolumeCopyArgs struct {
    // New name for the target
    Name string
}

type StoragePoolVolumeMoveArgs struct {
    StoragePoolVolumeCopyArgs
}
```

Reply:
> CopyStoragePoolVolume(pool string, source ContainerServer, sourcePool string, volume api.StorageVolume, args *StoragePoolVolumeCopyArgs) (op *RemoteOperation, err error)

Do you want this to be `api.StorageVolume` and in both cases?

Reply:
type StoragePoolVolumeCopyArgs struct {
    // New name for the target
    Name string
}

type StoragePoolVolumeMoveArgs struct {
    StoragePoolVolumeCopyArgs
}

Ah, I see.